web-ui/src/main/resources/catalog/views/default/templates/footer.html

```diff
@@ -40,7 +40,7 @@
       data-ng-click="getPermalink('', socialMediaLink)"
       title="{{'permalink' | translate}}"><i class="fa fa-fw fa-link"></i></a>
   </li>
-  <li>
+  <li ng-show="showSocialMediaLink">
```

Comment:
RSS is not a "social link" IMO

Reply:
No, not really, but we also wanted to be able to hide it.